packages/proxy/lib/http/index.ts

```diff
@@ -429,10 +430,13 @@ export class Http {
     }
   }
 
-  reset () {
+  reset (options: { fullReset?: boolean } = {}) {
```

Comment:
How would someone know if something belongs in a `fullReset`? Would it be better to just explicitly call out reseting prequests (i.e. `resetPrerequests: boolean`)?

Reply:
Updated: https://github.com/cypress-io/cypress/pull/28517/commits/956dd0aa92c8e7cdb0ee1284be92bdeeb52426a3